tests/functional/test_proxy_behaviour.py

```diff
@@ -144,7 +148,7 @@ def setup_rate_limit_proxy(setup_session):
 @pytest.mark.rate_limit
 @pytest.mark.apmspii_627
```

Comment:
what is the apimspii_627? could it be renamed to something more descriptive or does it need to stay as it is?

Reply:
I don't know of it doing anything other than providing the ability to run tests marked with that marker.

Reply:
okay, that's fine. I think it would be worth creating a low priority tech debt ticket for oddly named markers to made more descriptive so it's immediately obvious and the code doesn't need to go back hunting through jira tickets